frontend/coprs_frontend/coprs/logic/complex_logic.py

```diff
@@ -798,13 +799,19 @@ def _delete_reason(cls, copr_chroots):
         # project owner. Or all of them for the same reason. Also all
         # architectures may be deleted by the project owner but on a different
         # day and therefore the remaining time may be different)
+        # See https://docs.pagure.org/copr.copr/developer_documentation/eol-logic.html
         reasons = {}
         for chroot in delete_chroots:
             reason_format = "{0} and will remain available for another {1} days"
-            reason = reason_format.format(
-                "disabled by a project owner" if chroot.is_active else "EOL",
-                chroot.delete_after_days,
-            )
+            reason = "EOL"
+            if chroot.is_active:
+                if chroot.deleted and chroot.delete_after is None:
```

Comment:
This `and chroot.delete_after is None` is eventually wrong;  because `.delete_after` is set to a proper value when user disables the chroot.  Going to drop it :)

Reply:
No no no, the condition is just reversed, there should be `delete_after is not None`.